internal/app/networkd/pkg/networkd/netconf.go

```diff
@@ -81,19 +83,19 @@ func buildOptions(device machine.Device) (name string, opts []nic.Option, err er
 	}
 
 	// if device.Bond.ARPIPTarget {
-	//	opts = append(opts, nic.WithUseCarrier(device.Bond.UseCarrier))
+	//	opts = append(opts, nic.WithARPIPTarget(device.Bond.ARPIPTarget))
 	//}
```

Comment:
commented intentionally?

Reply:
Yeah. There's a couple attributes that use netlink.NestedAttribute which i didn't want to deal with yet.

Reply:
cool